spec/features/account_manager/deleting_an_entry_spec.rb

```diff
@@ -1,11 +1,9 @@
 require 'spec_helper'
 
-feature 'account manager deleting an entry' do
+feature 'deleting an entry' do
```

Comment:
hm, am i missing something? role name is only part of the failed example path. but maybe it's more consistent since, as far as I remember, the existing feature specs did not include the role name either.